app/app/local_settings.py.dist

```diff
@@ -117,6 +118,10 @@ GITTER_TOKEN = '' #https://developer.gitter.im/docs/authentication
 # optional: only needed if you slack things
 SLACK_TOKEN = 'TODO'
 
+# Rollbar tokens
+ROLLBAR_CLIENT_TOKEN = os.environ.get('ROLLBAR_CLIENT_TOKEN', '')
```

Comment:
youll want to make a similar update to the production local settings for this.

Reply:
whats best practice for local development?  should we all have our own rollbar tokens?

Reply:
For local development, I was leaving that up to the individual.  If the tokens (`ROLLBAR_SERVER_TOKEN`/`ROLLBAR_CLIENT_TOKEN`) aren't provided, rollbar is disabled by default.  If that will suffice, I'll update the documentation to reflect that.

Reply:
works for me